File: backend/src/modules/health/health.routes.js
import { Router } from 'express';

const router = Router();

router.get('/', (req, res) => {
  res.json({
    success: true,
    data: { status: 'ok', service: 'reconnect-ai-backend', timestamp: new Date().toISOString() },
    error: null,
    meta: {},
  });
});

export default router;

File: ai-service/app/api/v1/health_routes.py
from fastapi import APIRouter
from datetime import datetime, timezone

router = APIRouter()


@router.get("/health")
async def health():
    return {
        "success": True,
        "data": {
            "status": "ok",
            "service": "reconnect-ai-service",
            "timestamp": datetime.now(timezone.utc).isoformat(),
        },
        "error": None,
    }
